book/cli/reth/stage/unwind.md

```diff
@@ -65,6 +65,117 @@ Database:
+          The interval in seconds at which to carry out periodic lookup queries, for the whole run of the program
+          
+          [default: 60]
+
+      --discovery.v5.bootstrap.lookup-interval <DISCOVERY_V5_bootstrap_lookup_interval>
+          The interval in seconds at which to carry out boost lookup queries, for a fixed number of times, at bootstrap
+          
+          [default: 5]
+
+      --discovery.v5.bootstrap.lookup-countdown <DISCOVERY_V5_bootstrap_lookup_countdown>
+          The number of times to carry out boost lookup queries at bootstrap
+          
+          [default: 100]
+
+      --trusted-peers <TRUSTED_PEERS>
+          Comma separated enode URLs of trusted peers for P2P connections.
+          
+          --trusted-peers enode://abcd@192.168.0.1:30303
+
+      --trusted-only
+          Connect only to trusted peers
+
+      --bootnodes <BOOTNODES>
+          Comma separated enode URLs for P2P discovery bootstrap.
+          
+          Will fall back to a network-specific default if not specified.
+
+      --peers-file <FILE>
+          The path to the known peers file. Connected peers are dumped to this file on nodes
+          shutdown, and read on startup. Cannot be used with `--no-persist-peers`.
+
+      --identity <IDENTITY>
+          Custom node identity
+          
+          [default: reth/<VERSION>-<SHA>/<ARCH>]
+
+      --p2p-secret-key <PATH>
+          Secret key to use for this node.
+          
+          This will also deterministically set the peer ID. If not specified, it will be set in the data dir for the chain being used.
+
+      --no-persist-peers
+          Do not persist peers.
+
+      --nat <NAT>
+          NAT resolution method (any|none|upnp|publicip|extip:\<IP\>)
+          
+          [default: any]
+
+      --addr <ADDR>
+          Network listening address
+          
+          [default: 0.0.0.0]
+
+      --port <PORT>
+          Network listening port
+          
+          [default: 30303]
+
+      --max-outbound-peers <MAX_OUTBOUND_PEERS>
+          Maximum number of outbound requests. default: 100
+
+      --max-inbound-peers <MAX_INBOUND_PEERS>
+          Maximum number of inbound requests. default: 30
+
+      --pooled-tx-response-soft-limit <BYTES>
+          Soft limit for the byte size of a `PooledTransactions` response on assembling a `GetPooledTransactions` request. Spec'd at 2 MiB.
+          
+          <https://github.com/ethereum/devp2p/blob/master/caps/eth.md#protocol-messages>.
+          
+          [default: 2097152]
+
+      --pooled-tx-pack-soft-limit <BYTES>
+          Default soft limit for the byte size of a `PooledTransactions` response on assembling a `GetPooledTransactions` request. This defaults to less than the [`SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE`], at 2 MiB, used when assembling a `PooledTransactions` response. Default is 128 KiB
+          
+          [default: 131072]
```

Comment:
we should open a follow up issue to make these actually helpful for end users, they look like rustdocs

Reply:
https://github.com/paradigmxyz/reth/issues/8060